### PR TITLE
Add Shop Features to Main Page

### PR DESCRIPTION
**Pull request by marinrika**
### Description
Added shop features to  the main page.

### Problem
Problem issued [here](https://github.com/HNY-Badger/ecommerce-app/issues/3).

### Solution
Added content to the page.

### Impact
Enhanced the appearance of the main page.

### CODE REVIEW

File: src/pages/MainPage/MainPage.tsx [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+import ShopDescription from '../../components/MainPage/ShopDescription/ShopDescription';
+import ShopFeatures from '../../components/MainPage/ShopFeatures/ShopFeatures';
 
 function MainPage() {
-  return <ShopDescription />;
+  return (
+    <>
+      <ShopDescription />
+      <ShopFeatures />
+    </>
```
Comment (HNY-Badger) [inline src/pages/MainPage/MainPage.tsx]: Не лучше ли будет это обернуть в main тег?

File: src/components/MainPage/ShopFeatures/ShopFeatures.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    description: 'Exclusively top-tier products sourced from globally acclaimed manufacturers',
+  },
+  {
+    icon: GoCreditCard,
+    title: 'Unbeatable prices',
+    description: 'With our flexible discount system, you won’t find better prices anywhere else',
+  },
+  {
+    icon: IoShieldCheckmarkOutline,
+    title: 'Service and warrantys',
```
Comment (HNY-Badger) [inline src/components/MainPage/ShopFeatures/ShopFeatures.tsx]: Опечатка. "Service and <ins>warranties</ins>"

File: src/pages/MainPage/MainPage.tsx (hunk 1 above)
Comment (marinrika) [reply src/pages/MainPage/MainPage.tsx]: Исправила

File: src/components/MainPage/ShopFeatures/ShopFeatures.tsx (hunk 2 above)
Comment (marinrika) [reply src/components/MainPage/ShopFeatures/ShopFeatures.tsx]: Исправила

**Review by HNY-Badger — APPROVED** [review]
Всё отлично

**Pull request merged**